release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -1337,8 +1332,21 @@ Support for Docker v2 registries is deprecated and is planned for removal in a f
 [id="ocp-release-note-bare-metal-hardware-bug-fixes_{context}"]
 === Bare Metal Hardware Provisioning
 
+* Before this update, when installing a dual-stack cluster on bare metal using installer-provisioned infrastructure, the installation failed because the Virtual Media URL was IPv4 instead of IPv6. As IPv4 was unreachable, the bootstrap failed on the virtual machine (VM) and cluster nodes were not created. With this release, when you install a dual-stack cluster on bare metal for installer-provisioned infrastructure it uses the Virtual Media URL IPv6 and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-60240[OCPBUGS-60240]) 
```

Comment:
```suggestion
* Before this update, when installing a dual-stack cluster on bare metal by using installer-provisioned infrastructure, the installation failed because the Virtual Media URL was IPv4 instead of IPv6. As IPv4 was unreachable, the bootstrap failed on the virtual machine (VM) and cluster nodes were not created. With this release, when you install a dual-stack cluster on bare metal for installer-provisioned infrastructure, the dual-stack cluster uses the Virtual Media URL IPv6 and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-60240[OCPBUGS-60240]) 
```